src/IB/IIMethod.cpp

```diff
@@ -4414,6 +4441,9 @@ IIMethod::getFromInput(Pointer<Database> db, bool /*is_from_restart*/)
     if (db->isBool("use_pressure_jump_conditions"))
         d_use_pressure_jump_conditions = db->getBool("use_pressure_jump_conditions");
 
+    if (db->isBool("use_Tan_velocity_correction"))
```

Comment:
I think the option name should have something about velocity interpolation in the name. Also, is there a clear geometrical difference between the two approaches? It might be clearer to use that in the option name than the name of the first author of the paper.

Reply:
You may need to consider different interface positions and orientations.